### PR TITLE
[TOPIC-BLE-LLCP] Bluetooth: controller: renaming of configuration files

### PR DESCRIPTION
**Pull request by kruithofa**
Rename the config files for babblesim tests and associated
test scripts

depends on PR #40289

Signed-off-by: Andries Kruithof <Andries.Kruithof@nordicsemi.no>

### CODE REVIEW

File: tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app/prj_dut.conf [hunk 1]
```diff
@@ -28,7 +28,4 @@ CONFIG_BT_CTLR_DATA_LENGTH_MAX=60
 
 # Enabling BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED requires BT_CTLR_ADVANCED_FEATURES
 CONFIG_BT_CTLR_ADVANCED_FEATURES=y
-CONFIG_BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED=y
-
-# LLCP Refactored controller does not support Advanced Scheduling yet
-CONFIG_BT_CTLR_SCHED_ADVANCED=n
+CONFIG_BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED=n
```
Comment (erbr-ot) [inline tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app/prj_dut.conf:31]: CONFIG_BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED no longer

**Review by erbr-ot — CHANGES_REQUESTED** [review]
CONFIG_BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED is no longer

File: tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app/prj_dut.conf (hunk 1 above)
Comment (kruithofa) [reply tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app/prj_dut.conf]: Good catch, and here I was thinking I caught them all :(

**Review by erbr-ot — APPROVED** [review]

**Comment by thoh-ot** [comment]
CI not good

**Comment by kruithofa** [comment]
> CI not good

we need to get PR #40289 in first, and then this needs rebasing

**Review by erbr-ot — APPROVED** [review]
LGTM

**Review by thoh-ot — APPROVED** [review]

**Pull request merged**